src/main/java/io/appium/java_client/pagefactory/AndroidFindBy.java

```diff
@@ -94,4 +94,11 @@
      * @return priority of the searching
      */
     int priority() default 0;
+
+    /**
+     * Its Android View Tag element value.
+     *
+     * @return a android view tag value to the target element
+     */
+    String androidViewTag() default "";
```

Comment:
isn't it the same as `tagName`?

Reply:
Hi, both does different work. There is a example for this in #1269.

Reply:
I still have doubts on that. AndroidFindBy(tagName) should work just fine. Is there a reason to create another synonym?

Reply:
@jayandran-Sampath As @mykola-mokhnach says this should work

```
@AndroidFindBy(tagName = "something")
```

Reply:
@saikrishna321 & @mykola-mokhnach sure i will check once from my side and see how it works and then we can close this PR if its not needed. 

Just to be clear, I did this PR to enable PageFactory Annotation option for Locator Strategy -  Android View Tag (Espresso only) & also i see TagName method points to default Selenium By.TagName.

Reply:
@mykola-mokhnach & @saikrishna321 Thanks both for clearing it.. Tried with hello world app, as you guys said its working with AndroidFindBy(TagName) locator itself.

Just a thought, Internally using Tagname annotation we will still depend on Core By.tagName Locator Stragey to make it work but having seperate androidViewTag annotation will actually represent MobileBy.AndroidView Locator Stragey so in future this independent tag will work if there is any change in finding the element in espresso. 

![image](https://user-images.githubusercontent.com/24522469/69795958-1da06f00-11f3-11ea-9900-741c349b2e88.png)

If still we feel it's not required, We can close this PR.

Reply:
@jayandran-Sampath Sorry for getting back late on this. I think we should not add one more locator strategy instead we should have test specific to `android view` tag.

Reply:
okay @SrinivasanTarget . i will close this PR. Thanks

Reply:
If this is your decision, documenting this functionality for users would be helpful. I cannot locate documentation for the tagName annotation.

According to the AndroidFindBy javado, tagName returns "A desired element tag".

TBH, I would have expected the annotation to be called viewTag for consistency, 'name' some times refers to accessibility or 'tag name' to me suggests class (tag) name. Since this query will only be supported on a single driver, this naming seems problematic/unintuitive. What does tagName do on UIAutomator2?

e.g. [findElementByName/By.Name clearly resolves to content-desc (accessibility)](https://stackoverflow.com/a/26989612)


e.g. in [these docs](https://appium.readthedocs.io/en/latest/en/commands/element/attributes/name/) a tag name also corresponds to an accessibility label